.github/workflows/sycl_nightly.yml

```diff
@@ -12,8 +12,8 @@ jobs:
     uses: ./.github/workflows/sycl_gen_test_matrix.yml
     with:
       # Restore once plugins are enabled in the build.
-      # lts_config: "hip_amdgpu;ocl_gen9;ocl_x64;l0_gen9;esimd_emu;cuda_aws;win_l0_gen12"
-      lts_config: "ocl_gen9;ocl_x64;l0_gen9;win_l0_gen12"
+      # lts_config: "hip_amdgpu;ocl_gen12;ocl_x64;l0_gen12;esimd_emu;cuda_aws;win_l0_gen12"
+      lts_config: "ocl_gen12;ocl_x64;l0_gen12;win_l0_gen12"
```

Comment:
Why don't we use `lin_intel` instead of all linux specific configurations?

Reply:
Because we want to test `XFAIL` behavior somewhere. So we use a "faster" fused task in pre-commit and perform the "longer" testing in nightly.